cmd/dp-filter-api/main.go

```diff
@@ -91,13 +85,12 @@ func main() {
 	// todo: remove config.DatasetAPIAuthToken when the DatasetAPI supports identity based auth.
 	datasetAPI := dataset.NewAPIClient(cfg.DatasetAPIURL, cfg.ServiceAuthToken, "")
 
-	observationStore := observation.NewStore(pool)
 	previewDatasets := preview.DatasetStore{Store: observationStore}
 	outputQueue := filterOutputQueue.CreateOutputQueue(producer.Output())
 
 	healthTicker := healthcheck.NewTicker(
 		cfg.HealthCheckInterval,
-		neo4jhealth.NewHealthCheckClient(pool),
+		//	observationStore.Healthcheck(),
```

Comment:
Do we want this commented out?

Reply:
will be in a coming PR, in order to healthcheck this in abstraction-land we'll need to completely change the way healthchecks work so felt a significantly difference change that's worth separating